<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:url(:/img/Backgroung_2.png);
border-radius: 0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Shadow">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>0</y>
     <width>331</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 95);
border-top-right-radius: 20px; 
border-bottom-right-radius: 20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDateEdit" name="StartDate">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>200</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Perpetua</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0, 0, 0, 50);  
color: rgb(255, 255, 255);  
border: 1px solid #888888;
border-radius: 5px; 
padding: 2px;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="date">
    <date>
     <year>2024</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QDateEdit" name="EndDate">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>320</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Perpetua</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0, 0, 0, 50);  
color: rgb(255, 255, 255);  
border: 1px solid #888888;
border-radius: 5px; 
padding: 2px;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="date">
    <date>
     <year>2024</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QLabel" name="SubTitleSD">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>150</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SF Pro Display</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Start Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="SubTitleED">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>270</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SF Pro Display</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>End Date</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DeviceName">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>80</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SF Pro Display</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
color:rgb(255, 255, 255);
border-bottom: 2px solid rgba(255, 255, 255, 100);
padding-bottom:7px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
   <property name="placeholderText">
    <string>Device Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Generate">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>570</y>
     <width>291</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SF Pro Display</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(65, 141, 237, 255), stop:1 rgba(22, 55, 229, 255));
    color: white;
    border-radius: 10px;
    text-shadow: 6px 6px 4px rgba(0, 0, 0, 1);
    transition: background-color 0.3s ease, transform 0.1s ease;
}

QPushButton:hover {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(22, 55, 229, 255), stop:1 rgba(65, 141, 237, 255));
    cursor: pointer;  /* Change cursor to pointer (hand icon) */
}

QPushButton:pressed {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(15, 30, 180, 255), stop:1 rgba(50, 100, 200, 255));
    transform: scale(0.95);
}
</string>
   </property>
   <property name="text">
    <string>Generate Graph</string>
   </property>
  </widget>
  <widget class="QLabel" name="Graph">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>921</width>
     <height>691</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/admin_start.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Legend">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>391</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/Transparan.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Exit">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>0</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SF Pro Display</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 0, 0, 255), stop:1 rgba(255, 47, 47, 255));
    color: white;
    border-radius: 0px;</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
